src/api/comment/content-types/comment/lifecycles.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
Strange, that this file shows as new, while we have it already in main 🤔

Reply:
Oh, I see, there are multiple conflicts. Please make sure you have updated your branch with the latest main